gem/styles/gem-malawi/auth/zathu.enhanced.scss

```diff
@@ -284,28 +286,32 @@ hr {
   margin-top: $spacer-multiple;
 }
 
+
+.Footer {
+  margin: $spacer-multiple 0 0;
+  padding: $spacer-multiple;
+  text-align: left;
+  p {
+    margin: 0;
+  }
+}
 .Menu {
   margin: 0;
-
   &-title {
     background: $colorPurple;
     margin-left: 0;
     margin-right: 0;
     text-align: center;
-
     h1 {
       color: #fff;
       font-size: 1.2em;
     }
   }
-
   &-list {
-    padding-top: $spacer-half;
+    padding-top: 0;
```

Comment:
is this not the same as not having padding?

Reply:
No quite. For Zathu yes, however we are overriding GE default styles inheritance elsewhere with every rule.